apps/rebar/src/rebar_dialyzer_format.erl

```diff
@@ -249,7 +260,9 @@ message_to_string({opaque_call, [M, F, Args, Culprit, OpaqueType]}) ->
     fmt("~!^The call ~!!~ts:~ts~ts~!^ breaks the opaqueness of the term~!!"
         " ~ts :: ~ts", [M, F, Args, Culprit, OpaqueType]);
 %%----- Warnings for concurrency errors --------------------
-message_to_string({race_condition, [M, F, Args, Reason]}) ->
+% race_conditions was deleted in OTP-25
+% https://github.com/erlang/otp/pull/5502
+message_to_string({race_conditions, [M, F, Args, Reason]}) ->
```

Comment:
Actually, looking at the original PR, the atom was _not_ plural. We need to keep it as-is for backwards compat https://github.com/erlang/otp/pull/5502/files#diff-14627674b25a85392c29df29ffc6934257ad6f8a2a502cdea3350a272236e883

Reply:
:+1: I dropped the commit that contained the change.